fofix/core/GameEngine.py

```diff
@@ -358,18 +358,24 @@ def __init__(self, config = None):
             log.warn("No stages\ folder found, forcing None setting for Animated Stage.")
             self.config.set("game","animated_stage_folder", "None") #MFH: force "None" when Stages folder can't be found
 
-
-
         try:
+            self.theme = None
             fp, pathname, description = imp.find_module("CustomTheme",[themepath])
-            theme = imp.load_module("CustomTheme", fp, pathname, description)
-            self.theme = theme.CustomTheme(themepath, themename)
+            try:
+                theme = imp.load_module("CustomTheme", fp, pathname, description)
+                self.theme = theme.CustomTheme(themepath, themename)
+            except ImportError as e:
+                # Unable to load module; log it, but continue with default Theme.
+                log.error("Failed to load CustomTheme.py from %s" % pathname)
+                #self.theme = Theme(themepath, themename)
         except ImportError:
-            self.theme = Theme(themepath, themename)
+            # Not found; but that's OK. We'll use the default Theme class.
```

Comment:
If you add the second `try` / `except` block, then log the error. Otherwise it's not useful.

Reply:
See above.